raiden/network/rpc/client.py

```diff
@@ -237,9 +229,6 @@ def block_number(self):
         """ Return the most recent block. """
         return self.web3.eth.blockNumber
 
-    def inject_stop_event(self, event):
```

Comment:
Why was this removed?

Reply:
It's in the commit's comment: the correct approach both for exception handling and stop time should remove the need for injecting/raising an exception in web3. Instead, whatever uses it should stop and exit gracefully.

Reply:
Okay, could you please remove the exception `RaidenShuttingDown`? This only exists because we were not relying on `GreenletExit`, and it is only raised from the web3 middleware.